### DISCUSSION TITLE
Actions errored in deno

### DISCUSSION THREAD

**Discussion by waptik** [wevm/frog-archived#318] [Ideas]
### Describe the bug

I'm trying to use the default template with deno. Given that hono runs perfectly on deno, i figured this library would also work but unfortunately that wasn't the case for actions.
<img width="661" alt="image" src="https://github.com/wevm/frog/assets/1687551/13e68876-9cef-41b0-b64f-1bf30d88c2f6">

### Link to Minimal Reproducible Example

https://github.com/waptik/frog-frame-example/tree/8aba2bb45b54e5456995ee6c5888ea606c91cff9

### Steps To Reproduce

1. Make sure you have [deno](https://deno.com) installed
2. clone the repo
3. cd inside the cloned repo
4. run `deno task dev` and visit http://localhost:8000/dev
5. click on any action button and see error logs inside your terminal

### Frog Version

0.8.1

### Check existing issues

- [X] I checked there isn't [already an issue](https://github.com/wevm/frog/issues) for the bug I encountered.

### Anything else?

I love for this library to have full support for deno without BYONM

**Comment by tmm** [comment]
Frog doesn't support Deno at the moment. Would love to support it, but likely requires some more rework.

**Reply by waptik** [reply]
Thanks for the response. I already ditched the idea a while ago.
I moved on to building with nextjs
